python/vineyard/core/tests/test_client_on_distributed_vineyard.py

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env python
```

Comment:
I cannot see if there are any cases that can be covered by this test file but cannot be covered by existing test cases.

Reply:
Actually, the main purpose of this test is to get the same data from different vineyard nodes.

Reply:
Move the test case to https://github.com/v6d-io/v6d/blob/main/python/vineyard/deploy/tests/test_distributed.py